### PR TITLE
WIP: Add Maintenance Window Content

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/8235

### CODE REVIEW

**Comment by keichwa** [comment]
General remark first: We should clearly differentiate between server and clients (proxy is a special client and should think about that one, too).

I'll now add my remarks in a way that they will make sense for the server.

File: modules/administration/pages/maintenance-window.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[[maintenance-window]]
+= Maintenance Window Checklist
+
+If you work on a system with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
```
Comment (keichwa) [inline modules/administration/pages/maintenance-window.adoc]:
```suggestion
If you work on a {productname} Server with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
```

File: modules/administration/pages/maintenance-window.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+This section contains a checklist for your maintenance window, with links to further information on performing each of the steps, if you need it.
+
+
+. Stop services.
```
Comment (keichwa) [inline modules/administration/pages/maintenance-window.adoc]: Do we want to single out special services such as "spacewalk services"?  Of course, if you work on a system it is always good if you stop as many services as possible.

File: modules/administration/pages/maintenance-window.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+. Check if the configuration is still correct, and apply the highstate.
```
Comment (keichwa) [inline modules/administration/pages/maintenance-window.adoc]: applying highstate only makes sense if you maintain the suma server with Salt tool.

File: modules/administration/pages/maintenance-window.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+. Apply the latest updates.
+See xref:upgrade:server-upgrade.adoc[].
+. Update to the latest service pack, if required.
+See xref:upgrade:sp-migration.adoc[].
+. Reboot the server.
```
Comment (keichwa) [inline modules/administration/pages/maintenance-window.adoc]: That's "safe", but not always necessary. You can even update (parts of) the kernel with live patching.

File: modules/administration/pages/maintenance-window.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+. Check if the configuration is still correct, and apply the highstate.
```
Comment (keichwa) [inline modules/administration/pages/maintenance-window.adoc]: about highstate, see above.

File: modules/administration/pages/maintenance-window.adoc (hunk 2 above)
Comment (Loquacity) [reply modules/administration/pages/maintenance-window.adoc]: The original had "SAP and databases", but when I thought about it, it's highly dependent on what you have running. Perhaps a general statement, though?

File: modules/administration/pages/maintenance-window.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/administration/pages/maintenance-window.adoc]: Agreed, but it's probably best to mention it.

**Pull request merged**